src/include/storage/page/page.h

```diff
@@ -14,54 +14,62 @@
 
 #include <cstring>
 #include <iostream>
+#include <vector>
 
 #include "common/config.h"
 #include "common/rwlatch.h"
 
 namespace bustub {
 
 /**
+ * @brief
+ *
+ * TODO(2024 tas) This entire class needs to be updated.
+ *
```

Comment:
We should rip out this entire file.

Reply:
Other parts of the system still rely on this file and we haven't ripped them out yet. We'll update this once we start making changes for P2